Doc/c-api/refcounting.rst

```diff
@@ -139,3 +144,36 @@ objects.
 
    Decrement the reference count for object *o*. A function version of :c:func:`Py_XDECREF`.
    It can be used for runtime dynamic embedding of Python.
+
+
+.. c:macro:: Py_SETREF(dst, src)
+
+   Macro safely decrementing the `dst` reference count and setting `dst` to
+   `src`.
```

Comment:
I tried to make it clear that Py_SETREF() and Py_XSETREF() are implemented as macros.